site/docs/try/releases.md

```diff
@@ -2,6 +2,15 @@
 
 See [Nessie Server upgrade notes](server-upgrade.md) for supported upgrade paths.
 
+## 0.43.0 Release (September 15, 2022)
+
+**Rolling upgrades from versions before 0.40.0 are not supported!**
+
+* Prepare for Nessie GC
+* Nessie export/import functionality
+* Use Graal 22.2-java17 for native images
+* Several test and build improvements
```

Comment:
Would it be worth mentioning key list lookup fixes? Those bugs seem to be quite possible in real applications.

Reply:
I made a few last minute touch-ups to #5174 based on reviews... should be mergeable tomorrow

Reply:
I too agree on merging #5174 and releasing with that fix.